javase/src/main/java/com/google/zxing/client/j2se/DecoderConfig.java

```diff
@@ -58,6 +58,10 @@
       description = "Only output one line per file, omitting the contents")
   boolean brief;
 
+  @Parameter(names = "--json",
```

Comment:
Does it make sense to offer something in JSON when --json and --brief are both set?